finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryResource.java

```diff
@@ -45,12 +47,16 @@ public Response getPropertiesForHost(@PathParam("hostname") String hostname) {
         manager.add(hostname, props);
         return Response.ok(props).build();
     }
-
+    // end::getPropertiesForHost[]
```

Comment:
- add blank line above this line so methods are spaced out properly

Reply:
Addressed.